sonner/build.gradle.kts

```diff
@@ -25,6 +25,20 @@ kotlin {
         }
         binaries.executable()
     }
+    js(IR) {
+        moduleName = "compose-sooner"
```

Comment:
Maybe we can add `-jscanvas` to pass the CI

Reply:
ok. Just doing it